### DISCUSSION TITLE
How to implement it in a project

### DISCUSSION THREAD

**Discussion by Anghelr1** [warting/android-signaturepad#199] [Q&A]
Howdy!
I've been looking at your bookstore and I really like the way you've done it.
Do you think you could share with us how to implement it?
I've been trying to implement it in a blank project for a while now, and I haven't figured out how to do it.

**Comment by warting** [comment]
This repo contains a sample app using it in different ways, please have a look at https://github.com/warting/android-signaturepad/tree/main/app

**Comment by drtyrell969** [comment]
My question is do I need to include the three libraries? core, pad and view? I use data binding. The override for onSigned fires the entire time the pad is being draw into, but never executes after the pen comes up. I think that's the purpose of that call right?
